core/src/main/scala/za/co/absa/spline/model/TempReference.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.model
+
+case class TempReference(id: String)
```

Comment:
The class is very specific to the implementation of expression converters. Can it be moved to a narrower scope of visibility? Also a sentence of scala doc would help to understand it's role.